runtime/common/src/main/java/edu/snu/nemo/runtime/common/data/HashRange.java

```diff
@@ -19,22 +19,23 @@
  * Descriptor for hash range.
  */
 public final class HashRange implements KeyRange<Integer> {
-  private static final HashRange ALL = new HashRange(0, Integer.MAX_VALUE);
-
+  private static final HashRange ALL = new HashRange(0, Integer.MAX_VALUE, false);
   private final int rangeBeginInclusive;
   private final int rangeEndExclusive;
+  private boolean isSkewed;
```

Comment:
Can we do without introducing this boolean here?
- It may be better to keep description (these key ranges are being used), and interpretation (these key ranges are skewed) separate
- This boolean must be updated every time the corresponding key range, or the definition of skewed key ranges change
- All KeyRange implementations should maintain this variable